src/components/nav-bar/index.tsx

```diff
@@ -1,157 +1,147 @@
+      buttons: [
+        { href: '/mailMerge', icon: <Email />, label: 'Emails' },
+        { href: '/user', icon: <Group />, label: 'Users' },
+        { href: '#', icon: <Equalizer />, label: 'Reports' },
+      ],
+    },
+  ];
+
   return (
-    <>
-      <Box height={'100vh'}>
-        <Box display={'flex'} justifyContent={'center'} p={2}>
+    <Box
+      height="100vh"
+      display="flex"
+      flexDirection="column"
+      width="33vh" // Fix width of the navbar
+      overflow="hidden" // Prevent any expanding overflow
+      borderRight="1px solid #ddd"
+    >
+      {/* Logo Section */}
+      <Box display="flex" justifyContent="center">
+        <Button href="/">
           <Image
-            className="rounded-md"
             alt="logo"
             src="/cropped-MOH-Logo-768x393.png"
-            width="180"
-            height="100"
-            // style={{ marginRight: '25px' }} // Add spacing between the image and links
+            width="220"
+            height="107"
+            className="rounded-md"
           />
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="./"
-            startIcon={<HomeIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Home
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation/add"
-            startIcon={<AddCircleIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Add Donation
-          </Button>
-        </Box>
-
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
-          <Button
-            href="/donation"
-            startIcon={<VolunteerActivismIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
-          >
-            Donations
-          </Button>
-        </Box>
+        </Button>
+      </Box>
 
-        <Box display={'flex'} justifyContent={'flex-start'} p={2}>
+      {/* Button Groups */}
+      {sections.map((section, index) => (
+        <Box key={index} width="100%">
+          {/* Section Header */}
           <Button
-            href="/donors"
-            startIcon={<BallotIcon />}
-            style={{
-              color: '#ff8a65',
-              fontSize: '25px',
-              textTransform: 'none',
-            }}
+            onClick={() => handleToggle(section.title)}
+            startIcon={section.icon} // Add icon here
+            endIcon={
+              expandedSection === section.title ? (
+                <ExpandLess />
+              ) : (
+                <ExpandMore />
+              )
+            }
```

Comment:
Is it possible to push these expand icons to the right of the navbar so they are all aligned vertically?

Reply:
I have pushed them to the right side to make it look aligned.